baybe/acquisition/base.py

```diff
@@ -72,17 +78,36 @@ def to_botorch(
         additional_params = {}
         if "model" in signature_params:
             additional_params["model"] = surrogate.to_botorch()
-        if "best_f" in signature_params:
-            additional_params["best_f"] = train_y.max().item()
         if "X_baseline" in signature_params:
             additional_params["X_baseline"] = to_tensor(train_x)
         if "mc_points" in signature_params:
             additional_params["mc_points"] = to_tensor(
                 self.get_integration_points(searchspace)  # type: ignore[attr-defined]
             )
 
-        params_dict.update(additional_params)
+        # Add acquisition objective / best observed value
+        match objective:
+            case SingleTargetObjective(NumericalTarget(mode=TargetMode.MIN)):
+                if "best_f" in signature_params:
+                    additional_params["best_f"] = train_y.min().item()
+
+                if issubclass(acqf_cls, bo_acqf.AnalyticAcquisitionFunction):
+                    additional_params["maximize"] = False
```

Comment:
`maximize` doesnt seem part of the base class or is it guaranteed otherwise that its accepted?

Reply:
True, but consequence of the current botorch class layout of `AnalyticAcquisitionFunction`. Will only break for `ScalarizedPosteriorMean`, which we don't offer so far. See [here](https://github.com/pytorch/botorch/discussions/2442#discussioncomment-10496523). In case they decide to change this, no action required on our side. Otherwise, we can add the necessary workaround once we actually support `ScalarizedPosteriorMean`.